ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -348,8 +327,8 @@ QtObject {
         if (!tx || tx.txType !== Constants.TransactionType.Send)
             return false
 
-        let res = root.lookupAddressObject(tx.sender)
-        if (!res || res.type !== RootStore.LookupType.Account || res.object.walletType == Constants.watchWalletType)
+        let res = SQUtils.ModelUtils.getByKey(root.accounts, "address", tx.sender)
```

Comment:
`lookupAddressObject` was checking 2 models. Isn't it necessary now?

Reply:
Please, correct me if I'm wrong but what I iterpret is that:
Before:
1. The `lookupAddressObject` could get `acountObject`, `sharedAddressesObject` or `null`.
2. Then here, if the result was or `null` or not `acountObject`, it returned `false`.

Whit my implementation:
1. We only call the `accounts` check, since it was the only meaningful case of the previous conditions, right?